pages/events/[id].tsx

```diff
@@ -0,0 +1,103 @@
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { useGetEventByIdQuery } from 'generated/graphql';
+import { Typography } from 'components/atoms/Typography/Typography';
+import { useState } from 'react';
+import { EventDetails } from 'components/templates/events/eventDetails';
+import { BudgetDetails } from 'components/templates/events/budgetDetails';
+
+const EventDetailsPage = () => {
+	const router = useRouter();
+	const id = router.query.id as string;
+	const { data, loading } = useGetEventByIdQuery({ variables: { id } });
+	type View = 'event' | 'budget';
+	const [view, setView] = useState<View>('event');
+	function RenderPage({ viewProps }: { viewProps: string }) {
```

Comment:
I think we should call this component more descriptive. What about _EventDetailsBudget_ and _EventDetalisSummary_

Reply:
Done